mitreattack/navlayers/README.md

```diff
@@ -247,6 +249,9 @@ t.to_svg(layerInit=lay, filepath="demo.svg")
 conf = SVGConfig()
 conf.load_from_file(filename="path/to/poster/config.json")
 
-t2 = ToSvg(domain='mobile', source='local', local='path/to/local/stix.json', config=conf)
+t2 = ToSvg(domain='mobile', source='local', resource='path/to/local/stix.json', config=conf)
 t2.to_svg(layerInit=lay, filepath="demo2.svg")
+
+t3 = ToSvg(domain='enterprise', source='remote', resource='workbench.instance.url', config=conf)
```

Comment:
It might be better to do:
```
workbench_url = "localhost:3000"
t3 = ToSvg(domain='enterprise', source='remote', resource=workbench_url, config=conf)
```
Just to make it a little clearer what it a typical value might be.

Reply:
Good thinking - that's quite a bit clearer.